agent/catalog_endpoint.go

```diff
@@ -356,16 +357,8 @@ func (s *HTTPHandlers) catalogServiceNodes(resp http.ResponseWriter, req *http.R
 		args.TagFilter = true
 	}
 
-	if _, ok := params["merge-central-config"]; ok {
-		args.MergeCentralConfig = true
-	}
```

Comment:
Should this be getting reverted too? It doesn't seem related to the url escape changes

Reply:
oh good catch! Probably a victim of my deletes